Content.Server/Light/EntitySystems/PoweredLightSystem.cs

```diff
@@ -103,6 +104,9 @@ private void OnInteractHand(EntityUid uid, PoweredLightComponent light, Interact
             if (bulbUid == null)
                 return;
 
+            // delay the interaction by 2 seconds
+            var curTime = _gameTiming.CurTime;
+
```

Comment:
It would be better for this to be placed right before the `if` statement that uses it. I'm not sure if the C# optimizer is smart enough to move the actual assignment into the if clause.